MonitorizareVot.xcodeproj/project.pbxproj

```diff
@@ -74,9 +74,12 @@
 		EF4BD3921D9D08DE0042BEC7 /* s.xcdatamodeld in Sources */ = {isa = PBXBuildFile; fileRef = EF4BD3901D9D08DE0042BEC7 /* s.xcdatamodeld */; };
 		EF4BD3A21D9D08DE0042BEC7 /* MonitorizareVotTests.m in Sources */ = {isa = PBXBuildFile; fileRef = EF4BD3A11D9D08DE0042BEC7 /* MonitorizareVotTests.m */; };
 		EF4BD3AD1D9D08DE0042BEC7 /* MonitorizareVotUITests.m in Sources */ = {isa = PBXBuildFile; fileRef = EF4BD3AC1D9D08DE0042BEC7 /* MonitorizareVotUITests.m */; };
-		EF903E351DA25E3100D5BC9C /* Judete.plist in Resources */ = {isa = PBXBuildFile; fileRef = EF903E341DA25E3100D5BC9C /* Judete.plist */; };
 		EFB9AD1A1DE9FDE0008C1035 /* DBSyncer.swift in Sources */ = {isa = PBXBuildFile; fileRef = EFB9AD191DE9FDE0008C1035 /* DBSyncer.swift */; };
 		EFEC82041DCF25B800610D79 /* QuestionCollectionViewCell.xib in Resources */ = {isa = PBXBuildFile; fileRef = EFEC82021DCF25B800610D79 /* QuestionCollectionViewCell.xib */; };
+		F14A46C522A3417800088D8D /* LocalPollingStationsPersistor.swift in Sources */ = {isa = PBXBuildFile; fileRef = F14A46C422A3417800088D8D /* LocalPollingStationsPersistor.swift */; };
+		F1ACBFAD22A3065900EE4E3A /* LocalConfiguration.xcconfig in Resources */ = {isa = PBXBuildFile; fileRef = F1ACBFAC22A3065900EE4E3A /* LocalConfiguration.xcconfig */; };
```

Comment:
LocalConfig should not be added to the project or to the repository. There's a README in the Config folder. You should remove it before accepting this PR

Reply:
Will remove for this PR, but this should have a more elegant solution. The file is in gitignore, but it's unexpected and cumbersome to ask contributors to remove it only to submit the PR.

Reply:
Well, not exactly. The LocalConfig doesn't need to be included in the project because it's included locally by the default xcconfig file, so you should never have it included in the project or git repo. That's the whole point of not sharing your own configuration with others, this being an open source project with public access. If you have a different solution, let us know. I've been using this solution for a while now and it always worked great